sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamingSymmetricHashJoinExec.scala

```diff
@@ -246,13 +244,14 @@ case class StreamingSymmetricHashJoinExec(
 
     //  Join one side input using the other side's buffered/state rows. Here is how it is done.
     //
-    //  - `leftJoiner.joinWith(rightJoiner)` generates all rows from matching new left input with
```

Comment:
I'm not familiar with this part, cc @zsxwing  @HeartSaVioR @xuanyuanking

Reply:
The comment seems to be just modified for replacing `leftJoiner.joinWith(rightJoiner)` with `leftSideJoiner.storeAndJoinWithOtherSide(rightSideJoiner)` and vice versa for right side. Other parts aren't modified.

Reply:
@cloud-fan , @HeartSaVioR - yes, this is just updating the comment, because there's no `leftJoiner`/`rightJoiner`/`joinWith` in the file, and the original author (https://github.com/apache/spark/pull/19271) should mean to refer to `leftSideJoiner`/`rightSideJoiner`/`storeAndJoinWithOtherSide`. I think it would make sense to be consistent  between code and comment here. This is anyway a minor change for comment only.

Reply:
I think the original PR just wants to use pseudocode to explain, either way is ok to me.